R/summarise_scores.R

```diff
@@ -60,13 +62,17 @@ summarise_scores <- function(scores,
                              by = "model",
                              across = NULL,
                              fun = mean,
+                             metrics,
```

Comment:
I'm not sure we strictly _need_ an additional argument given that users can specify the metrics in `score()`.

_If_ we decide to have it then I'd vote for making the default `metrics = get_metrics(scores)`.

Reply:
What does `get_metric` do if the input isn't a score object? That was the rub here. 

I only added this so I can could use `summarise_scores` on more general objects and save introducing more code.

More generally I also quite like the idea that naive users can reduce their score object at this point as many likely won't know to manipulate the metric list going into score regardless of docs. 

All of above being said I think this is a new issue to discuss.

Reply:
Maybe it makes sense not to use `summarise_scores()` here (at least for now). 

`summarise_scores()` ultimately is a glorified one-liner and the actual work happens here: 

```
scores <- scores[, lapply(.SD, fun, ...),
    by = c(by),
    .SDcols = colnames(scores) %like% paste(metrics, collapse = "|")
  ]
```

Then we could discuss the `metrics` argument in a separate issue/PR.

Reply:
Sure